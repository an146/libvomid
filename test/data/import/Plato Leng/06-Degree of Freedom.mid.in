../../../../test/mid/Plato Leng/06-Degree of Freedom.mid