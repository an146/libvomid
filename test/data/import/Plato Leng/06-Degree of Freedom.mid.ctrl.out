Tempo:0 461538 20160 375000 185520 382166 185760 389610 186000 400000 186240 408163 186480 416667 186720 428571 186960 437956 187200 447761 187440 461538 187680 472441 187920 483871 188160 500000 235200 461538
Time Signature:97920 515 120960 516
Track 0: Name:
Track 0: Volume:127
Track 0: Pan:64
Track 0: Balance:
Track 0: Expression:
Track 1: Name:
Track 1: Volume:102
Track 1: Pan:64
Track 1: Balance:
Track 1: Expression:
Track 2: Name:
Track 2: Volume:107
Track 2: Pan:64
Track 2: Balance:
Track 2: Expression:
Track 3: Name:
Track 3: Volume:90
Track 3: Pan:64
Track 3: Balance:
Track 3: Expression:
Track 4: Name:
Track 4: Volume:90
Track 4: Pan:64
Track 4: Balance:
Track 4: Expression:
Track 5: Name:
Track 5: Volume:119
Track 5: Pan:64
Track 5: Balance:
Track 5: Expression:
Track 6: Name:
Track 6: Volume:114
Track 6: Pan:64
Track 6: Balance:
Track 6: Expression:
Channel 0: Program:0 35
Channel 0: Pitch Wheel:
Channel 1: Program:0 30
Channel 1: Pitch Wheel:
Channel 2: Program:0 30
Channel 2: Pitch Wheel:
Channel 3: Program:0 25
Channel 3: Pitch Wheel:
Channel 4: Program:0 25
Channel 4: Pitch Wheel:
Channel 5: Program:0 42
Channel 5: Pitch Wheel:
Channel 6: Program:
Channel 6: Pitch Wheel:
Channel 7: Program:
Channel 7: Pitch Wheel:
Channel 8: Program:
Channel 8: Pitch Wheel:
Channel 9: Program:
Channel 9: Pitch Wheel:
Channel 10: Program:
Channel 10: Pitch Wheel:
Channel 11: Program:
Channel 11: Pitch Wheel:
Channel 12: Program:
Channel 12: Pitch Wheel:
Channel 13: Program:
Channel 13: Pitch Wheel:
Channel 14: Program:
Channel 14: Pitch Wheel:
Channel 15: Program:
Channel 15: Pitch Wheel:
